runtime/lua/vim/lsp.lua

```diff
@@ -588,6 +588,12 @@ function lsp.start_client(config)
       -- process is not alive then the server should exit (see exit notification)
       -- its process.
       processId = uv.getpid();
+      -- Information about the client
+      -- since 3.15.0
+      clientInfo = {
+        name = 'Neovim',
+        version = vim.fn.matchstr(vim.fn.execute('version'), [[NVIM \zs[^\n]\+]]) or vim.Nil
```

Comment:
Same with version. I feel like we should have a lua style guide like the C style guide. I don't even know what the defacto standard is 😆

Reply:
lol it's ironic you bring this up because I looked through the file to see if there was a pattern of single or double, and I saw both so just picked single. I can totally change them.

Reply:
Haha I think I have a selection bias because I only saw double quotes, but looking back I saw single.